Frontend/ui-customer/src/Flow.purs

```diff
@@ -1197,11 +1219,28 @@ homeScreenFlow = do
         homeScreenFlow
       else do
         PlaceName placeDetails <- getPlaceName lat lon gateAddress
+        let currentLocationItem = Just (dummyLocationListState{
+                              prefixImageUrl = "ny_ic_recent_search,https://assets.juspay.in/nammayatri/images/user/ny_ic_recent_search.png"
+                            , postfixImageUrl = "ny_ic_fav,https://assets.juspay.in/nammayatri/images/user/ny_ic_fav.png"
+                            , title = (fromMaybe "" ((split (Pattern ",") (placeDetails.formattedAddress)) !! 0))
+                            , subTitle = (drop ((fromMaybe 0 (indexOf (Pattern ",") (placeDetails.formattedAddress))) + 2) (placeDetails.formattedAddress))
+                            , placeId = placeDetails.placeId
+                            , lat = Just (if state.props.currentStage /= ConfirmingLocation then lat 
+                                        else if (state.props.isSource == Just true) then state.props.sourceLat
```

Comment:
`isSource` can be Just false also sometimes? if yes then refactor that to make it `Nothing` and use isJust in if-else conditions